wiring/inc/spark_wiring_ble.h

```diff
@@ -301,12 +306,6 @@ class BLEClass {
     BLEClass();
     ~BLEClass();
 
-    /* Initialize BLE stack and add pre-defined local attributes. */
-    int on(void);
-
-    /* Turn BLE off. */
-    int off(void);
```

Comment:
Sorry for the confusion! Please KEEP `on()` and `off()` defined. I only removed them because in our simple example apps, there should be no need to call `on()` since BLE will be on by default. Big 👍 that users may want to turn the radio off in some low power situations. On the other hand, if it simplifies things to not have to implement this now, we could save on and off for a later release.